src/Model/User.php

```diff
@@ -96,6 +96,36 @@ class User
 
 	private static $owner;
 
+	private static function welcome_text(): string
+	{
+		return Strings::deindent(DI::l10n()->t('
+		The login details are as follows:
+
+		Site Location:	%1$s
+		Login Name:		%2$s
+		Password:		%3$s
```

Comment:
Don't use tabs as styling elements in i18n, just use spaces instead. Translators will very likely overseen them and you are making assumptions about the styling that won't match in other languages/locales making it harder for the translators.

```suggestion
		Site Location: %1$s
		Login Name: %2$s
		Password: %3$s
```

Reply:
I did not add them, just copied the original text, but I agree and good points - will remove them.

Reply:
@mfxa I'm sorry, I wasn't referring to you personally. My comment was meant to be general.